src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -1618,6 +1618,28 @@ HMICapabilitiesImpl::GetDefaultInitializedCapabilities() const {
   return default_initialized_capabilities_;
 }
 
+void HMICapabilitiesImpl::RemoveFromDefaultInitializedCapabilities(
+    hmi_apis::FunctionID::eType requested_interface) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (app_mngr_.IsHMICooperating()) {
+    return;
```

Comment:
@ydementieiev Add some meaningful log message here. Try to make a silent return nowhere

Reply:
@AByzhynar [done](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/0d60c4e63814fd53aa0e6c9e0f26893e9587efca#diff-f67d217da60dca297da36f9d5dcbeac3R1626)